Kickstarter-iOS/Views/Controllers/ChangePasswordViewController.swift

```diff
@@ -53,6 +54,9 @@ final class ChangePasswordViewController: UIViewController, MessageBannerViewCon
     _ = self.scrollView
       |> \.alwaysBounceVertical .~ true
 
+    _ = self.stackView
```

Comment:
Should we also set `preservesSuperviewLayoutMargins` to `false`?

Looking at this in SIMulator, the horizontal padding looks smaller than `12pts`. Using visual inspector actually shows `8pts`.

Reply:
Hmm I think the default is false. I'll investigate 🕵️‍♀️

Reply:
Ok the issue is that `preservesSuperviewLayoutMargins` needs to be set to `true` on all the `stackView`'s arranged subviews. This is because they're all container views which hold the label and the text field.

Reply:
That's pretty unfortunate. I guess this is where the concept of using `UITableViewCell` and its `contentView` would be helpful (in a sense that we'd only need to set the margins for the content view).

Good find Isabel!